index.html

```diff
@@ -0,0 +1,723 @@
+          <p>
+            Marc Kamionkowski of Johns Hopkins University, an early-universe
+            expert who was not part of the team, said, “This is huge, as big as
+            it gets.”
+          </p>
+          <p>
+            He continued, “This is a signal from the very earliest universe,
+            sending a telegram encoded in gravitational waves.”
+          </p>
+          <p>
+            The ripples manifested themselves as faint spiral patterns in a bath
+            of microwave radiation that permeates space and preserves a picture
+            of the universe when it was 380,000 years old and as hot as the
+            surface of the sun.
+          </p>
+          <p>
+            Dr. Kovac and his collaborators, working in an experiment known as
+            Bicep, for Background Imaging of Cosmic Extragalactic Polarization,
+            reported their results in a scientific briefing at the Center for
+            Astrophysics here on Monday and in a set of papers submitted to The
+            Astrophysical Journal.
+          </p>
+        </div>
+      </div>
+
+      <h2>The Theory of Inflation</h2>
+
+      <div class="figcaption">
+        Astronomers have found evidence to support the theory of inflation,
+        which explains how the universe expanded so uniformly and so quickly in
+        the instant after the Big Bang 13.8 billion years ago.
+      </div>
+
+      <div class="theoryInflation">
+        <div class="row1">
+          <div class="column">
+            <p>
+              THE UNIVERSE is just under 14 billion years old. From our position
+              in the Milky Way galaxy, we can observe a sphere that is now about
+              92 billion light-years across. But there's a mystery. Wherever we
+              look, the universe has an even temperature.
+            </p>
+          </div>
+          <div class="column">
+            <p>
+              NOT ENOUGH TIME The universe is not old enough for light to have
+              traveled the vast distance from one side of the universe to the
+              other, and there has not been enough time for scattered patches of
+              hot and cold to mix into an even temperature.
+            </p>
+          </div>
+          <div class="column">
+            <p>
+              DISTANT COFFEE At a smaller scale, imagine using a telescope to
+              look a mile in one direction. You see a coffee cup, and from the
+              amount of steam, you can estimate its temperature and how much it
+              has cooled.
+            </p>
+          </div>
+          <div class="column">
+            <p>
+              COFFEE EVERYWHERE Now turn around and look a mile in the other
+              direction. You see a similar coffee cup, at exactly the same
+              temperature. Coincidence? Maybe. But if you see a similar cup in
+              every direction, you might want to look for another explanation.
+            </p>
```

Comment:
Improve styling on the `Theory of Inflation section
- [ ] This section has its elements widely spaced and covering the entire page. It shouldn't. Instead, it should resemble that of the original page.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|![theory-of-inflation](https://user-images.githubusercontent.com/22345961/108399729-7eb6c700-722b-11eb-8a57-a86d8bd027b0.png)|![theory-of-inflation-clone](https://user-images.githubusercontent.com/22345961/108399747-82e2e480-722b-11eb-97ac-ee9a381ad6f5.png)|